src/components/data-table-v2/_data-table-v2-action.scss

```diff
@@ -162,6 +162,35 @@
     }
   }
 
+  .#{$prefix}--batch-edit-status {
+    display: flex;
+    align-items: center;
+    flex: 1;
+    color: $ui-01;
+  }
+
+  .#{$prefix}--batch-edit-summary {
+    padding-left: rem(16px);
+
+    &__cancel {
```

Comment:
Could we flatten out the styles for this for code consistency? Definitely understand it's more typing 😓 haha

Reply:
Thank you for reviewing @joshblack! FWIW it doesn’t cause nested selector in built version of CSS (Instead `.bx--batch-edit-summary__cancel` selector will be created), but I can change the coding style if you’d like. Let me know!

Reply:
Right, it's more so just because of the coding style. The rationale being that we should try and match CSS as closely as possible if we ever want to switch between tools.

Reply:
I see - Made an update!

Reply:
Hmm, seems like it might be still showing up in the diff for cancel and save, potentially?

Reply:
@joshblack Here’s the change in case it helps: https://github.com/IBM/carbon-components/pull/1020/commits/c945ac3019e5308629dc6a15c28dcc37b929518d

Reply:
Got it - Thank you for spotting @joshblack!